x-pack/plugins/file_upload/server/telemetry/mappings.ts

```diff
@@ -7,13 +7,13 @@
 import { SavedObjectsType } from 'src/core/server';
 import { TELEMETRY_DOC_ID } from './telemetry';
 
-export const mlTelemetryMappingsType: SavedObjectsType = {
+export const telemetryMappingsType: SavedObjectsType = {
   name: TELEMETRY_DOC_ID,
   hidden: false,
   namespaceType: 'agnostic',
   mappings: {
     properties: {
-      file_data_visualizer: {
+      file_upload: {
         properties: {
           index_creation_count: {
```

Comment:
Testing, after ES restart, I get a `500` error from the `/api/file_upload/import` endpoint when I hit the Import button - `{"statusCode":500,"error":"Internal Server Error","message":"Cannot read property 'index_creation_count' of undefined"}`. Is there some initialization step for the telemetry missing?

Reply:
Could be the result of an extra level of embedding in the telemetry object:

![image](https://user-images.githubusercontent.com/1728976/106303970-022a5d00-6218-11eb-8beb-25d86270dc27.png)

Reply:
@peteharverson I was able to recreate the problem you were seeing and have resolved it.

The problem was that I had not changed `TELEMETRY_DOC_ID`. This resulted in errors where the new file_upload telemetry logic would query for telemetry saved object. Since the id did not change, there was a collision where it would find a saved object from the old ml telemetry code. This old saved object would be run through the new code and would fail because it was the wrong shape. Updating `TELEMETRY_DOC_ID` to a unique name resolves the problem. Please try testing again with the latest changes.